command/agent/fs_endpoint_test.go

```diff
@@ -342,6 +342,45 @@ func TestHTTP_FS_Cat(t *testing.T) {
 }
 
 func TestHTTP_FS_Stream(t *testing.T) {
```

Comment:
Given that this tests the non-default case, I would call the follow=false in the function name?  So maybe 

```suggestion
func TestHTTP_FS_Stream_NoFollow(t *testing.T) {
```